examples/large_files/test/huge_file_tests.ts

```diff
@@ -0,0 +1,56 @@
+import { describe } from '@jest/globals';
+import { please, Test } from 'azle/test/jest';
+import { execSync } from 'child_process';
+import { rm } from 'fs-extra';
+import { join } from 'path';
+
+import { Unit } from '../../../scripts/file_generator';
+import { generateTestFileOfSize } from './generate_test_files';
+import { getAutoGeneratedFileName, verifyUpload } from './tests';
+
+const hugeAutoGenAutoUploadTests: [number, Unit][] = [[2, 'GiB']];
+
+export function hugeFilesTests(origin: string): Test {
+    return () => {
+        please(
+            'remove all other auto generated files so there is room for huge files',
+            async () => {
+                await rm(join('assets', 'auto'), {
+                    recursive: true,
+                    force: true
+                });
+            }
+        );
+
+        describe.each(hugeAutoGenAutoUploadTests)(
```

Comment:
Rename to `hugeAutoGenAutoUploadTests` to something not ending in `Tests`

Reply:
done

Reply:
Plural

Reply:
done